docker_tests/test_docker_compose_quick_start.py

```diff
@@ -154,9 +175,16 @@ def test_trigger_dag_and_wait_for_result(tmp_path_factory, monkeypatch):
     except Exception:
         print(f"Current working directory: {os.getcwd()}")
         run_command(["docker", "version"])
-        run_command(["docker-compose", "version"])
+        run_command([*compose_command, "version"])
         run_command(["docker", "ps"])
-        run_command(["docker-compose", "logs"])
+        run_command([*compose_command, "logs"])
+        for container in ALL_CONTAINERS:
+            print("Health check for {container}")
+            result = run_command(
+                ["docker", "inspect", "--format", "{{json .State}}", container], return_output=True
+            )
+            pprint(result)
         raise
     finally:
-        run_command(["docker-compose", "down", "--volumes"])
+        if not os.environ.get("SKIP_DOCKER_COMPOSE_DELETION"):
```

Comment:
It would be awesome to add a Breeze cli flag/option as well, so you could specify something like: `breeze testing docker-compose-tests --skip-deletion`, not a requirement for this PR though.